### PR TITLE
WSTEAMA-390 - Update React Component Standards docs with component migration guidelines

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves JIRA [[390](https://jira.dev.bbc.co.uk/browse/WSTEAMA-390)]

Overall changes
======
Adding instructions from [https://paper.dropbox.com/doc/Migrate-MostRead-from-Legacy--B81Po_DXuITHAPaXjbGUxJ~3Ag-0FHywUTsf55Brd9VaYUxs](https://paper.dropbox.com/doc/Migrate-MostRead-from-Legacy--B81Po_DXuITHAPaXjbGUxJ~3Ag-0FHywUTsf55Brd9VaYUxs) to our React Component Standards docs. 

[Read it here!](https://wsteama-390-update-react-component-st--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/coding-standards-migrating-from-legacy--page)

Code changes
======

- A new `stories.mdx` file created and populated with migration instructions and guidelines.
- A link to the doc added from the standardisation section of the React Components doc. 

Testing
======

- Code changes ran with `yarn storybook` and viewed via [http://localhost:9001/?path=/story/coding-standards-migrating-from-legacy--page](http://localhost:9001/?path=/story/coding-standards-migrating-from-legacy--page) to check for typos!!

Helpful Links
======

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: docs/Coding-Standards/Migrating-From-Legacy.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,413 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Coding Standards/Migrating From Legacy" />
```
Comment (karinathomasbbc) [inline docs/Coding-Standards/Migrating-From-Legacy.stories.mdx]:
```suggestion
<Meta title="Coding Standards/Migrating Legacy Components" />
```

**Review by karinathomasbbc — APPROVED** [review]
Niiiice! This looks really good in Storybook, and now becomes a place where we can add more standards if we need to!

Thanks for writing and reviewing these docs!

**Comment by karinathomasbbc** [comment]
Just a quick thought - should we link to this document somewhere from the React Components doc? https://wsteama-390-update-react-component-st--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/coding-standards-react-components--page - there is a section about standardisation, which might be a good place to put this

**Review by amoore108 — APPROVED** [review]
Looks great, nice job!

**Pull request merged**